crates/utils/src/utils/validation.rs

```diff
@@ -22,6 +22,8 @@ const BIO_MAX_LENGTH: usize = 300;
 const ALT_TEXT_MAX_LENGTH: usize = 300;
 const SITE_NAME_MAX_LENGTH: usize = 20;
 const SITE_NAME_MIN_LENGTH: usize = 1;
+const TAGLINE_CONTENT_MIN_LENGTH: usize = 1;
+const TAGLINE_CONTENT_MAX_LENGTH: usize = 50000;
```

Comment:
This seems extremely long, same as max post body. I think 200 or so would be enough.

Reply:
Btw only `is_valid_post_title()` calls `trim()` before length check. Would probably make sense to `trim` inside of `min_length_check` and `min_length_check` so that it applies to all strings.

cc @dessalines

Reply:
Hexbear uses taglines far longer than 200 characters. Lemmygrad as well.

Reply:
I took 50000 which is the default max body length. Seems like too much indeed. Maybe 500 chars?

Reply:
I checked hexbear and lemmygrad taglines manually, longest one I noticed is 1170 chars. That seems extremely long but if we want to support it, we can set a limit of 1500 or 2000 chars.

Reply:
Agreed @dessalines. Shall we keep it as is, or remove the max limit completely?

Reply:
As is seems fine to me. You've externalized it so we can always tweak it later easily.